api/comments/serializers.py

```diff
@@ -0,0 +1,155 @@
+from datetime import datetime
+from rest_framework import serializers as ser
+from framework.guid.model import Guid
+from framework.auth.core import Auth
+from website.project.model import Node, Comment
+from rest_framework.exceptions import ValidationError, PermissionDenied
+from api.base.utils import absolute_reverse
+from api.base.serializers import (JSONAPISerializer,
+                                  JSONAPIHyperlinkedRelatedField,
+                                  JSONAPIHyperlinkedGuidRelatedField,
+                                  JSONAPIHyperlinkedIdentityField,
+                                  IDField, TypeField, LinksField)
+
+
+class CommentReport():
```

Comment:
Is the purpose of this class just to provide structure for the input data?

Two small comments/ possibilities:
1. If you stick with the class, it's general good practice for Python2 classes to inherit from `object`. Otherwise some stuff gets weird.
2. A more lightweight (memory-efficient) way to structure this data would be with a `namedtuple`:

```
from collections import namedtuple

comment_tuple = collections.namedtuple('comment_tuple', 
    ['user_id', 'category', 'text'])

g = comment_tuple("bob", "boring", "text")
print g.text
```